docker-compose.yml

```diff
@@ -1,30 +1,34 @@
 version: "3"
 services:
-    posteewebhook:
+    postee:
+        container_name: postee
         build:
             context: .
             dockerfile: Dockerfile
         ports:
             - 8082:8082
         volumes:
             - bolt:/server/database
-            - config:/server
+            - config:/config
         environment:
-            - "POSTEE_CFG=/server/cfg.yaml"
+            - "POSTEE_CFG=/config/cfg.yaml"
             - "POSTEE_DEBUG=ON"
     posteeui:
+        depends_on:
+            - postee
+        container_name: postee-ui
         build:
             context: .
             dockerfile: Dockerfile.ui
         ports:
             - 8001:8001
         volumes:
             - bolt:/server/database
-            - config:/server
+            - config:/config-files
```

Comment:
What's the difference between the `/config` mount and `/config-files` mount? Can we keep them the same to avoid confusion?

Reply:
When I was testing I had a confusion about the orginal mount of postee, and then postee-ui. I will keep the original names.